std/range/package.d

```diff
@@ -9223,3 +9223,5 @@ if (Rs.length > 1 &&
     m.popFront;
     assert(m.empty);
 }
+
+package T* addressOf(T)(ref T val) { return &val; }
```

Comment:
Also, what does adding a new function have to do with property enforcement? Should it be a separate PR? The rest of this PR looks good to me.